tests/classify_test.py

```diff
@@ -128,3 +128,12 @@ def test_application_directories(in_tmpdir, no_empty_path):
     # Should be application with extra directories
     ret = classify_import('testing', application_directories=('.', 'tests'))
     assert ret is ImportType.APPLICATION
+
+
+def test_application_modules(in_tmpdir, no_empty_path):
+    # Should be classified 3rd party without argument
+    ret = classify_import('testing')
+    assert ret is ImportType.THIRD_PARTY
+    # Should be application with application_modules override
+    ret = classify_import('testing', application_modules=('testing',))
```

Comment:
if someone passed `__future__` here it would cause a `SyntaxError` -- I don't think that one should be overridable

Reply:
Nice catch. I've changed it to raise `ValueError`